wpengine-phpcompat.php

```diff
@@ -267,93 +268,176 @@ function settings_page() {
+								</td>
+							</tr>
+							<tr>
+								<th scope="row"><label for="active_plugins"><?php _e( 'Plugin / Theme Status', 'php-compatibility-checker' ); ?></label></th>
+								<td>
+									<fieldset>
+										<label><input type="radio" name="active_plugins" value="yes" <?php checked( $only_active, 'yes', true ); ?> /> <?php _e( 'Only scan active plugins and themes', 'php-compatibility-checker' ); ?></label><br>
+										<label><input type="radio" name="active_plugins" value="no" <?php checked( $only_active, 'no', true ); ?> /> <?php _e( 'Scan all plugins and themes', 'php-compatibility-checker' ); ?></label>
+									</fieldset>
+								</td>
+							</tr>
+							<tr>
+								<th scope="row"></th>
+									<td>
+										<div class="wpe-pcc-run-scan">
+											<input name="run" id="runButton" type="button" value="<?php _e( 'Scan site', 'php-compatibility-checker' ); ?>" class="button-secondary" />
+											<span style="display:none; visibility:visible;" class="spinner wpe-pcc-spinner"></span>
+										</div> <!-- /wpe-pcc-run-scan -->
+									</td>
+								</th>
+							</tr>
+						</tbody>
+					</table>
+				</div> <!-- /wpe-pcc-scan-options -->
+
+				<?php /* Scan results */ ?>
+				<div class="wpe-pcc-results" style="display:none;">
+					<hr>
+					<h2><?php printf( 'Scan Results for PHP <span class="wpe-pcc-test-version">' . $test_version . '</span> Compatibility' ); ?></h2>
+
+					<?php /* Download report */ ?>
+					<div class="wpe-pcc-download-report" style="display:none;">
+						<a id="downloadReport" class="button-primary" href="#"><span class="dashicons dashicons-download"></span> <?php _e( 'Download Report', 'php-compatibility-checker' ); ?></a>
+						<a class="wpe-pcc-clear-results" name="run" id="cleanupButton"><?php _e( 'Clear results', 'php-compatibility-checker' ); ?></a>
+						<?php /* Toggle developer mode */ ?>
+						<label class="wpe-pcc-developer-mode">
+							<input type="checkbox" id="developermode" name="developermode" value="yes" />
+							<?php _e( 'View results as raw text', 'php-compatibility-checker' ); ?>
+						</label>
+						<hr>
+					</div> <!-- /wpe-pcc-download-report -->
+
+					<?php /* Progress bar */ ?>
+					<div style="display:none;" id="wpe-progress">
+						<p><?php printf( '<strong>Scan progress</strong> - <span id="wpe-progress-count"></span> <span id="wpe-progress-active"></span>' ); ?></p>
+
+						<div id="progressbar"></div>
+					</div>
+
+					<?php /* Area for pretty results. */ ?>
+					<div id="wpe-pcc-standardMode"></div>
+
+					<?php /* Area for developer results. */ ?>
+					<div style="display:none;" id="developerMode">
+						<textarea readonly="readonly" id="testResults"></textarea>
+					</div>
+
+					<p class="wpe-pcc-attention"><?php _e( '<strong>Attention:</strong> Not all errors are show-stoppers. <a target="_blank" href="' . $url_get_hosting . '">Test this site in PHP7</a> to see if just works!', 'php-compatibility-checker' ); ?></p>
```

Comment:
A few notes here: 

> "Test this site in PHP7to see if just works!"

I think this is missing an "it". 

This line is also throwing an error with phpcs: 
```
FILE: /home/travis/build/wpengine/phpcompat/wpengine-phpcompat.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 355 | ERROR | The $text arg must be a single string literal, not
     |       | "'<strong>Attention:</strong> Not all errors are
     |       | show-stoppers. <a target="_blank"
     |       | href="'.$url_get_hosting.'">Test this site in PHP7</a>
     |       | to see if just works!'".
----------------------------------------------------------------------
```

WordPress wants you to use printf with these, so it can be translated without the PHP being processed. For example: 

```
printf( _e('<strong>Attention:</strong> Not all errors are show-stoppers. <a target="_blank" href="%s">Test this site in PHP7</a> to see if just works!', 'php-compatibility-checker' ), $url_get_hosting );
```